builtin/common/misc_helpers.lua

```diff
@@ -279,6 +279,15 @@ function core.wrap_text(text, max_length, as_table)
 	return as_table and result or table.concat(result, '\n')
 end
 
+
+function core.setting_get_pos(name)
+	local value = core.settings:get(name)
```

Comment:
this is not supposed to be here, merging mistake

Reply:
But its the only way to get access to it from the CSM API, right?

Reply:
If you have to then duplicate it in `builtin/client/`, not in some random other place.

Reply:
Is it OK now?